docs/written/trove/lists.scrbl

```diff
@@ -1623,13 +1623,104 @@ end
 }
 
 @list-method["member"]
+
+Returns true if the @pyret{elt} is located somewhere in the list.
+
+@examples{
+check:
+  [list: 'a', 'b', 'c'].member('a') is true
+  [list: 'a', 'b', 'c'].member( 1 ) is false
+  [list: ].member('a') is false
+end
+}
+
 @list-method["append"]
+
```

Comment:
Nevermind. There instead should be a documentation that lists have `_plus` which is appending